source/dem/insertion.cc

```diff
@@ -79,12 +79,12 @@ Insertion<dim>::assign_particle_properties(
                    ->particle_sizes[particle_counter]);
       double density =
         physical_properties.density_particle[current_inserting_particle_type];
-      double vel_x        = dem_parameters.insertion_info.vel_x;
-      double vel_y        = dem_parameters.insertion_info.vel_y;
-      double vel_z        = dem_parameters.insertion_info.vel_z;
-      double omega_x      = dem_parameters.insertion_info.omega_x;
-      double omega_y      = dem_parameters.insertion_info.omega_y;
-      double omega_z      = dem_parameters.insertion_info.omega_z;
+      double vel_x        = dem_parameters.insertion_info.initial_vel[0];
+      double vel_y        = dem_parameters.insertion_info.initial_vel[1];
+      double vel_z        = dem_parameters.insertion_info.initial_vel[2];
+      double omega_x      = dem_parameters.insertion_info.initial_ang_vel[0];
```

Comment:
I like the new parameter name, but I think I would rather keep `dem_parameters.insertion_info.omega[0]`, because the rest of the code uses omega. Otherwise it is all clear to me :)

Reply:
Ah, and you don't need `initial`, since this is insertion info

Reply:
I think the <initial> is useful. This way, there's not confusion with the velocity at a given time step. 

I can do initial_omega. I agree, omega is more coherent with the rest of code.

What do you think?

Reply:
In general, I tend to prefer long meaningful names instead of confusing abbreviations, but in this case I found it slightly redundant, hence, not necessary. This is not a strong opinion, though. So this is really up to you :)